packages/react-devtools-shell/src/devtools.js

```diff
@@ -17,8 +17,17 @@ const {contentDocument, contentWindow} = iframe;
 // Helps with positioning Overlay UI.
 contentWindow.__REACT_DEVTOOLS_TARGET_WINDOW__ = window;
 
+// Initialize the backend and frontend early so that we have the messages that we
+// need during initial render
 initializeBackend(contentWindow);
 
+// Activate the backend only once the DevTools frontend Store has been initialized.
+// Otherwise the Store may miss important initial tree op codes.
+// activateBackend(contentWindow);
+const DevTools = initializeFrontend(contentWindow);
+
+activateBackend(contentWindow);
```

Comment:
Can you add a comment explaining why we activate _after_ `initializeFrontend` now?

Reply:
Also if this change of ordering is important, we should update the instructions here to mention it:
https://github.com/facebook/react/tree/main/packages/react-devtools-inline

Reply:
Fair, although the overall sequence did change. Before, we were initializing the frontend and backend inside the `initDevTools` callback:
https://github.com/facebook/react/blob/64931821a9df262f2b6475404d025bcff97c3e0a/packages/react-devtools-shell/src/devtools.js#L45-L70

But now we're doing it outside, before we inject:
https://github.com/lunaruan/react/blob/340d25482c959894cf31214cea4fbbbcab2fe3c6/packages/react-devtools-shell/src/devtools.js#L20-L28

Probably so we don't miss console logs that happen during injection? Just wondering if there's any note we should add to the docs (README) for others. (If there isn't, that's okay. I just wanted to check.)